ci/priv-integration.sh

```diff
@@ -106,6 +108,19 @@ img_commit2=$(ostree --repo=${repo} rev-parse ostree/container/image/${imgref})
 test "${img_commit}" = "${img_commit2}"
 echo "ok deploy derived container identical revs"
 
+ostree-ext-cli container image deploy --sysroot "${sysroot}" \
+        --stateroot "${stateroot}" --imgref ostree-unverified-image:"${derived_img_dir}"
+imgref=$(ostree refs --repo=${repo} ostree/container/image | head -1)
+img_commit=$(ostree --repo=${repo} rev-parse ostree/container/image/${imgref})
+ostree-ext-cli container image remove --repo "${repo}" "${derived_img_dir}"
+
+ostree-ext-cli container image deploy --sysroot "${sysroot}" \
```

Comment:
Then here I'd do:
```
rm -rf /var/tmp/derived.dir
```
or so to avoid "leaking" double disk space usage across the rest of the test.

I think we can also drop the re-deploy and test of identical commit ID, because that's very unlikely to break just with the `dir` transport.
.